paper.tex

```diff
@@ -118,10 +118,8 @@ \subsubsection{Blind Signature}
 \section{Trading Model}
 In the following, we describe the data trading process in detail. To participate a data marketplace, data providers and consumers have to register first. Then data provider can launch its product on the marketplace. Once a product is launched, it is searchable and could be traded afterward. The whole trading and refunding process is defined in smart contracts which are easily traceable and irreversible.
 
-In our game theory model there are two kinds of players. One is data provider, and the other one is data buyer. Data provider would receive subscription fee for buying data each time the data is transmitted to data buyer.
-The data buyer will keep buying data from data povider, only when the data sold by the data provider with high enough accuracy, $P_r$. We deonte this accuracy threshold with $\xi$.
-Once the accuracy is lower than $\xi$, the data buyers would consider this data provider as a low quality data provider, and reject to buy the data from this data provider.
-
+Data provider would receive subscription fee for buying data each time the data is transmitted to consumer; the rest unpaid subscription fee would be stored in smart contract. The consumer will keep buying data from data povider, only when the data sold by the data provider with high enough accuracy, which we called these data as decent data.
```

Comment:
The usage of "the rest unpaid subscription" sounds weird. Can you refine the wording?

Reply:
how about I refine it into just "unpaid subscription fee"